nautobot/extras/tests/test_jobs.py

```diff
@@ -192,6 +193,28 @@ def test_no_field_order(self):
 <br><span class="helptext">Commit changes to the database (uncheck for a dry-run)</span></td></tr>""",
         )
 
+    def test_no_field_order_inherited_variable(self):
+        """
+        Job test without field_order with a variable inherited from the base class
+        """
+        module = "test_no_field_order"
+        name = "TestDefaultFieldOrderWithInheritance"
+        job_class = get_job(f"local/{module}/{name}")
+        form = job_class().as_form()
+        self.assertHTMLEqual(
+            form.as_table(),
+            """<tr><th><label for="id_testvar1">Testvar1:</label></th><td>
+            <input type="text" name="testvar1" class="form-control form-control" required placeholder="None" id="id_testvar1">
+            <br><span class="helptext">This var should come before any vars defined in subclasses</span></td></tr><tr><th>
+            <label for="id_b_testvar2">B testvar2:</label></th><td>
+            <input type="text" name="b_testvar2" class="form-control form-control" required placeholder="None" id="id_b_testvar2">
+            <br><span class="helptext">This var should be second</span></td></tr>\n<tr><th><label for="id_a_testvar3">A testvar3:</label></th><td>
+            <input type="text" name="a_testvar3" class="form-control form-control" required placeholder="None" id="id_a_testvar3">
+            <br><span class="helptext">This var should be third</span></td></tr>\n<tr><th><label for="id__commit">Commit changes:</label></th><td>
+            <input type="checkbox" name="_commit" placeholder="Commit changes" id="id__commit" checked><br><span class="helptext">
+            Commit changes to the database (uncheck for a dry-run)</span></td></tr>""",
+        )
```

Comment:
I get the intent here but being as this is comparing a block of rendered HTML, I feel like this test will be very fragile and might require updating often if we move things around.  Is there a way we can test for something we know should always be there without expecting the exact identical code?